app/upgrades/v7/upgrades.go

```diff
@@ -9,7 +9,7 @@ import (
 	icahosttypes "github.com/cosmos/ibc-go/v4/modules/apps/27-interchain-accounts/host/types"
 	icatypes "github.com/cosmos/ibc-go/v4/modules/apps/27-interchain-accounts/types"
 
-	"github.com/cosmos/gaia/v10/app/keepers"
+	"github.com/cosmos/gaia/v11/app/keepers"
```

Comment:
Same as above, this is `app/upgrades/v7` but you're importing the latest keeper.

Will this be cleaned up at a later time?